chassis/configuration/configuration/basic/default.environment.properties

```diff
@@ -406,4 +406,11 @@ chassis.engine.e1.reporting.session.comparison.decision.maker.warning.deviation.
 # F.e. threshold of 0.8 mean that values for current session should differ from baseline not more than +-80% to avoid errors
 chassis.engine.e1.reporting.session.comparison.decision.maker.fatal.deviation.threshold=0.8
 
+### Section for properties which is going to be used during configuration construction
```

Comment:
please, put it higher in this file. before string
# begin: following section is used for docu generation - Default setup for monitoring metrics